fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/NumericArithmetic.java

```diff
@@ -1136,21 +1136,10 @@ public static Expression mathE() {
     public static Expression truncate(DecimalV3Literal first, IntegerLiteral second) {
         if (first.getValue().compareTo(BigDecimal.ZERO) == 0) {
             return first;
+        } else if (first.getValue().compareTo(BigDecimal.ZERO) == -1) {
```

Comment:
```suggestion
        } else if (first.getValue().compareTo(BigDecimal.ZERO) < 0) {
```

Reply:
the function compareTo only return 0 -1 and 1, so I think -1 is acceptable for this situation

Reply:
you wrong. https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html#compareTo-T-

Reply:
https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html
maybe you can check bigDecimal Type implementation for compareTo

Reply:
changed to aligning interface definition